cli/Cargo.toml

```diff
@@ -188,6 +188,8 @@ walkdir.workspace = true
 weak-table.workspace = true
 zip = { workspace = true, features = ["deflate-flate2"] }
 zstd.workspace = true
+swc_html = "26.0.0"
```

Comment:
Discussed offline - we could switch this out for `lol_html` instead. We might do it in the future, but for now we're gonna try `swc_html`.